src/status_im/contexts/wallet/common/activity_tab/events.cljs

```diff
@@ -0,0 +1,45 @@
+(ns status-im.contexts.wallet.common.activity-tab.events
```

Comment:
These events were slightly changed and  moved to the activity-tab events, previously they were general